examples/hosts/hosts-sample/README.md

```diff
@@ -1,39 +1,62 @@
 # @fluid-internal/hosts-sample
 
-The Fluid loader is all that is needed to load any Fluid document. This example walks through all the steps to
-create, initialize, and then make use of the loader. And does so in a literate programming like style to provide
-more detail on each line of the code.
+This example demonstrates how to host Fluid components in a web application.
+The Fluid loader is needed to load a Fluid document. The example walks through all the steps to
+create, initialize, and then make use of the loader in under 200 lines of code.
+And does so in a literate programming-like style to provide more detail on each line of the code.
 
-There are other packages which provide simple APIs to make use of the loader. For example @fluidframework/base-host.
-These may be better starting options when integrating the loader into your own code. But this example will show all
-the steps needed to create and use the Fluid loader. And it still comes in under 200 lines
-of code.
+<!-- AUTO-GENERATED-CONTENT:START (GET_STARTED:tinylicious=true) -->
+<!-- The getting started instructions are automatically generated.
+To update them, edit md-magic.config.js in the root of the repo, then run npm run readme:update -->
 
-## Build steps
+## Getting Started
 
-Getting up and running is simple.
+You can run this example using the following steps:
 
-```bash
-npm install
-npm start
-```
+1. Run `npm install` and `npm run build:fast -- --nolint` from the `FluidFramework` root directory.
+   a. For an even faster build, you can add the package name to the build command, like this:
+   `npm run build:fast -- --nolint @fluid-internal/hosts-sample`
+1. In a separate terminal, start a Tinylicious server by following the instructions in [Tinylicious](../../../server/tinylicious).
+1. Run `npm run start` from this directory (examples/hosts/hosts-sample) and open <http://localhost:8080> in a web browser to see the app running.
+ <!-- AUTO-GENERATED-CONTENT:END -->
+
+## Files
```

Comment:
It might be good to include a short demo walkthrough somewhere in here that people should try before diving into implementation, e.g. try clicking the '+' button, try clicking the reload button, try major/minor version changes, etc.

Or include some pointers for usage in the HTML file itself (maybe this is the more "literate" approach).

I am also not exactly sure how to create a new document (looks like there's a magic hash?) - maybe would be good to make a note of that too.

Reply:
Of course, working on it, still tbd :) Just wanted to share early to get some feedback.

Reply:
Sounds good, I assumed as much :)  Just pulled your branch and realized I didn't know what I was doing :-P

Reply:
btw, the magic hash was there before my PR. And yet as this overhaul is quite substantial it's an opportunity to revisit all aspects of the sample.

Reply:
Resolved.